src/views/static/Status.vue

```diff
@@ -87,6 +87,14 @@ export default {
           isAlive: 'Waiting...',
           changelog: 'https://github.com/AEGEE/discounts/blob/master/CHANGELOG.md'
         },
+        network: {
+          roundTrip: null,
+          version: '-',
```

Comment:
I assume we still need to put the actual version numbers here

Reply:
It does some IT magic, but it works

Reply:
it's replaced in the function that fetches the Github stuff